jolly-jellyfish/src/page_maker/templates/page_maker/webpage_detail.html

```diff
@@ -0,0 +1,22 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+<div><h1>{{ webpage.user_title }}</h1></div>
+<div>
+  {% for comment in comments %}
+  <div>
+    <p>{{ comment.author.username }}: {{ comment.title }}</p>
+    <p>{{ comment.content }}</p>
+  </div>
+  {% endfor %}
+</div>
+<div>
+  <form method="POST" enctype="multipart/form-data" action="{% url 'comment-create' pagename=webpage.name %}">
+    {% csrf_token %}
+      <div>
+        {{ comment_form.as_p }}
+        <button type="submit">Add comment</button>
+      </div>
+    </form>
```

Comment:
```suggestion
  <form method="POST" enctype="multipart/form-data" action="{% url 'comment-create' pagename=webpage.name %}">
    {% csrf_token %}
    <div>
      {{ comment_form.as_p }}
      <button type="submit">Add comment</button>
    </div>
  </form>
```
Indentation adjusted